release notes/v0.37.0.md

```diff
@@ -0,0 +1,61 @@
+k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring, but it's just because we are preparing some incredible things in upcoming releases!
```

Comment:
```suggestion
k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring, ~~but it's just because we are preparing some incredible things in upcoming releases!~~
```
I'm not sure we want to do this promise. :smile:

Reply:
I prefer not to make promises :)

Reply:
Okay :+1: